gcc/rust/ast/rust-ast.cc

```diff
@@ -1273,6 +1273,10 @@ ExprStmtWithBlock::as_string () const
     {
       indent_spaces (enter);
       str += expr->as_string ();
+      if (semicolon_followed)
+	{
+	  str += ";";
+	}
```

Comment:
very small GNU style nitpick
```suggestion
	  str += ";";
```